### ISSUE TITLE
Contact page

### ISSUE THREAD

**Issue by kwcooper** [GCCR/GCCR.github.io#27]
I think it best that we pull the same page contact footers and add a dedicated contact page. This way, we can put the relative addresses for who folks should get in touch with, and the different pathways people should take to meet their needs (general inquiries, survey questions, membership, country heads, etc.)

@mariaveldhuizen would you mind bringing this up during the next meeting to get an idea for what they may want. Probably@ to figure out who the contacts should be? 

@cbouy can you start thinking about how this would best integrate into the existing schema of the site? I'm happy to build the page, but just want to cover my bases on not throwing anything else off. We have an existing link in the nav bar already for contact that directs to the form I built at the bottom of the page. We can re-purpose this link. Then add content when @mariaveldhuizen sends it to us. 

Or none of these things if we feel this isn't a good idea haha

**Comment by cbouy** [comment]
Yep good idea that would probably help in prioritizing the emails that we receive.

We would just have to remove the `{% include contact.html %}` from the default layout, change the link in data/navigation.yml from `#contact` to `/contact/`, and create the contact.html page in the root folder.

**Comment by mariaveldhuizen** [comment]
oh god, i think i never saw this. 
Yes, i absolutely think we need to have separate contact forms on each page or something along those lines. 

And no contact form on the "join" page because people keep missing the actual membership form.

**Comment by kwcooper** [comment]
While I'm revisiting the issues, did you ever hear back from the LT on this @mariaveldhuizen?
